radix-transactions/src/builder/manifest_builder.rs

```diff
@@ -95,13 +96,21 @@ pub struct NewSymbols {
     pub new_address_id: Option<u32>,
 }
 
-impl ManifestBuilder {
-    /// Starts a new transaction builder.
+pub type ManifestV1Builder = ManifestBuilder<InstructionV1>;
+
+impl ManifestBuilder<InstructionV1> {
+    /// Starts a new manifest v1 builder.
     pub fn new() -> Self {
+        Self::new_v1()
+    }
+
+    /// Call as `ManifestBuilder::new_v1()`
+    pub fn new_v1() -> Self {
```

Comment:
Not sure if `new_v1` makes sense since "v1" is already captured by the `InstructionV1` in the type?

Reply:
It’s so that you can do ‘ManifestBuilder::new_v1()’ rather than needing to fully specify ‘ManifestBuilder::<InstructionV1>::X()’ and it also gives parity to new_v2.

Ideally new wouldn’t exist, or wouldn’t exist just on v1, but it needs to map to v1 for backwards compatibility to save potentially breaking lots of dapp code.